scripts/test

```diff
@@ -44,7 +44,7 @@ compose() {
   elif [ "${debug}" = "true" ]; then
     docker-compose -f docker-compose.yaml -f docker-compose.test.yaml -f docker-compose.test.watch.yaml -f docker-compose.test.debug.yaml -p "${service}-test" $@
   else
-    docker-compose -f docker-compose.yaml -f docker-compose.test.yaml -p "${service}-test" --rm $@
+    docker-compose -f docker-compose.yaml -f docker-compose.test.yaml -p "${service}-test" $@
```

Comment:
Why remove this?  It stops a build up of redundant test containers on the host

Reply:
Ah got ya, it's in the wrong place.

Should be here I think

![image](https://user-images.githubusercontent.com/42574977/212736105-88e1cf75-ec68-407c-a565-d96e1aefe73d.png)

Reply:
Ah got ya, it's in the wrong place.

Should be here I think

![image](https://user-images.githubusercontent.com/42574977/212736335-137b3c94-6ef3-47e8-a9af-8a6fc2f9591e.png)